experiments.md

```diff
@@ -34,35 +34,37 @@ mkdir -p ${DATA_DIR}
 Select the dataset scale. This is roughly the storage size in GB.
 ```bash
 export SCALE=100  # For full-scale experiments presented in the paper.
```

Comment:
Could you move the comment to be the first line (so it describes the whole block)?